modules/dynamic-apiserver/main.go

```diff
@@ -17,10 +17,15 @@
 package main
 
 import (
-	"github.com/kuberlogic/kuberlogic/modules/dynamic-apiserver/cmd"
 	"os"
+
+	"github.com/kuberlogic/kuberlogic/modules/dynamic-apiserver/cmd"
+)
+
+var (
+	ver string
 )
 
 func main() {
-	cmd.Main(os.Args[1:])
+	cmd.Main(os.Args[1:], ver)
```

Comment:
Redudant, ldflags could works for any files

Reply:
fixed